src/taraxad/CMakeLists.txt

```diff
@@ -4,5 +4,5 @@ target_link_libraries(taraxad PRIVATE app_base)
 set(TARAXAD_INSTALL_DIR "/usr/local/bin" CACHE PATH "Aplication install directory")
 set(TARAXAD_CONF_INSTALL_DIR "/etc/taraxad" CACHE PATH "Aplication configuration install directory")
 
-install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/taraxad DESTINATION ${TARAXAD_INSTALL_DIR} COMPONENT taraxad)
+install(PROGRAMS ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/taraxad DESTINATION ${TARAXAD_INSTALL_DIR} COMPONENT taraxad)
```

Comment:
is this change somehow related to conan stuff ?

Reply:
Yeah, now all binaries will be at the <project_build_folder>/bin. So we need to change the way how we find bin files.

This is one of conan's requirements, as well as "specifying the build type" (Debug/Release/RelWithDebInfo)